lib/licensee/project_files/license_file.rb

```diff
@@ -34,11 +34,12 @@ class LicenseFile < Licensee::ProjectFiles::ProjectFile
         /#{COPYING_REGEX}-/                          => 0.65,  # COPYING-MIT
```

Comment:
What if instead, we changed the `-` on lines 33 and 34 to `[\-_] and just made the existing regex slightly more flexible?

Reply:
Also to note, I believe the regexes in lines 33 and 34 should be preceded by a `\A` to ensure it starts the file name.